src/scss/components/card/icon-cards/_icon-card.scss

```
@@ -54,7 +53,7 @@ $tag-height: 70px;
 
     .su-card__contents {
       @include modular-spacing('padding', 4 4 4 4);
```

```suggestion
      @include modular-spacing('padding', 4);
```
The 2nd part can take 1 to 4 numbers (or null) so just this would add padding for all 4 sides.